devops/openshift/api-deploy.yml

```diff
@@ -50,6 +50,14 @@ objects:
                   value: ${IS_FULLTIME_ALLOWED}
                 - name: POSTGRES_HOST
                   value: "${DB_SERVICE}"
+                - name: MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT
+                  value: "${MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT}"
+                - name: MAXIMUM_IDLE_TIME_FOR_WARNING_SUPPORTING_USER
+                  value: "${MAXIMUM_IDLE_TIME_FOR_WARNING_SUPPORTING_USER}"
+                - name: MAXIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION
+                  value: "${MAXIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION}"
+                - name: MAXIMUM_IDLE_TIME_FOR_WARNING_AEST
+                  value: "${MAXIMUM_IDLE_TIME_FOR_WARNING_AEST}"
```

Comment:
The parameters should be added to the `parameters` section at the end of the file. See others in the same file for an example.

Reply:
I just checked and the assignment is also missing in the `api-deploy`.
https://github.com/bcgov/SIMS/blob/f8fb5ca53590eac6aa78274d114cccdca2e24680/devops/Makefile#L391-L413